NEWS

```diff
@@ -1,3 +1,40 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new binary, ignition-validate, for validating Ignition configs
+    - Add overwrite field to files, directories, and links for deleting
+      preexisting items at the node's path
+    - Add options field to raid for specifying arbitrary mdadm options
+    - Add append field to files for appending to preexisting files
+    - Add support for specifying custom certificate authorities to use when
+      fetching objects over HTTPS
+
+  Changes
+
+    - Validate that partition labels don't contain colons, as sgdisk will
+      silently truncate the label
+    - Remove validate flag from Ignition
+    - Warn when the mode for a file or directory is unset
+    - Make HTTP fetching retry log at info level to make logs show up in console
+
+  Bug Fixes
+
+    - Fix issue where unspecified fields in an appended config could "unset"
+      fields specified in a config earlier in the chain
+    - Use timeouts specified in a config to fetch configs being appended or
+      replacing the current config
+    - Fetch S3 objects from the current AWS partition when running in EC2,
+      instead of always fetching from the default AWS partition
```

Comment:
Amazon calls it the `public AWS partition`.

Is this different from the change in 0.20.1?